.github/workflows/codespell.yml

```diff
@@ -0,0 +1,12 @@
+# https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
```

Comment:
Why is this comment needed?  Other workflow files don't have it.

Reply:
I copied it from ruff.yml which has it, but actually both these workflow files don't use much of what's documented there.

Will remove from both.